src/Nowruz/modules/general/components/SearchDropdown/index.tsx

```diff
@@ -150,6 +151,7 @@ export const SearchDropdown: React.FC<SelectProps> = ({
             indicatorSeparator: () => ({ display: 'none' }),
           }}
           {...props}
+          aria-labelledby="searchDropdown"

```

Comment:
Here also the 'area-labelledby' is fixed and it means that if we have multiple searchDropDown in a form, the 'area-labelledby' won't be unique .